charts/rhio/values.yaml

```diff
@@ -0,0 +1,43 @@
+replicaCount: 1
+
+image:
+  repository: ""
```

Comment:
I'm not sure what address should go here?

Reply:
you can have a look how version is generated in the github action.
There is `./version.sh "${{ env.REGISTRY }}/${{ vars.DOCKER_IMAGE_NAME }}"` statement. if you follow in version.sh, it is set to `repository` and `tag` during helm chart build.

Reply:
so you can leave repository empty as it is now.